content/guides/hosting/hosting-options/self-managed/kubernetes-operator/operator-airgapped.md

```diff
@@ -247,4 +247,4 @@ You must split the certificates into multiple entries in the `customCACerts` sec
 You can turn off auto-updates from the W&B console. Reach out to your W&B team for any questions on the supported versions. Also, note that W&B supports platform versions released in last 6 months. W&B recommends performing periodic upgrades. 
 
 ### Does the deployment work if the environment has no connection to public repositories?
-As long as you have enabled the `airgapped: true` configuration, the Kubernetes operator does not attempt to reach public repositories. The Kubernetes operator attempts to use your internal resources.
\ No newline at end of file
+As long as you have enabled the `airgapped: true` configuration, the Kubernetes operator does not attempt to reach public repositories. The Kubernetes operator attempts to use your internal resources.
```

Comment:
I think another PR in flight updates this sentence and I gave a suggested edit. Please keep this in sync.